### DISCUSSION TITLE
Sync root of remote location to a local folder

### DISCUSSION THREAD

**Discussion by pdelancie** [iterate-ch/cyberduck#16183] [Q&A]
I'm trying to perform a Synchronize operation where the target is the root directory at a remote location (see browser at right in screenshot) and the source is a folder on my local machine (at left of screenshot). But I don't see a way to actually select the remote root itself so that the Synchronize operation will sync the entire remote folder contents to the source. Instead I can only select individual folders within the target, and sync those to the corresponding folders on my local machine. I'm used to doing this easily in FileZilla Pro, but I don't see how to do it in CyberDuck. 

How do I either (a) select the root in the browser, or (b) synchronize to the remote root some other way?

Thank you! 

<img width="1199" alt="Screen Shot 2024-07-25 at 8 03 09 PM" src="https://github.com/user-attachments/assets/5f6ed696-78e1-4156-a05c-7d7cec20f95b">

**Comment by tatze96** [comment]
You can select the root for the synchronization feature either by right-clicking on white space within the root and selecting *Synchronize* within the appearing context menu or by choosing the same option within the application menu without having anything selected within the mounted root.

However, during our tests, we discovered that root folders are currently not synchronizable due to the bug described in #16191. We'll post any news on the solution there. So, the only way to currently synchronize or transfer the entire remote folder contents is to perform a manual file transfer.
